src/findings/utils.py

```diff
@@ -30,46 +30,47 @@ class FindingCategory(enum.Enum):
     FIELD_TYPE_CHANGE = 10
     FIELD_ONEOF_REMOVAL = 11
     FIELD_ONEOF_ADDITION = 12
+    FIELD_PROTO3_OPTIONAL_CHANGE = 13
```

Comment:
Not for this PR.
If these are often changing, it may make sense to spread the sections out a bit, so adding a new value doesn't require a complete re-number.
E.g. Start `# Field annotations` at 20, `# Messages` at 30, etc...

Reply:
Sure, SGTM!

Reply:
Done.